scripts/alert_emails/template.html

```diff
@@ -11,6 +11,15 @@
     <!--<![endif]-->
     <meta name="viewport" content="width=device-width">
     <meta name="x-apple-disable-message-reformatting">
+    <style>
+        .donate-button {
+            font-size: 10px;
+        }
+        @media (min-width: 600px) {
+            .donate-button {
+                font-size: 15px;
+            }
+        }
```

Comment:
Media queries and classes don't work consistently across email clients, I think is safer to just set the font-size to 10px in the `style` prop of the `a` element below